methods.py

```diff
@@ -41,3 +52,61 @@ def compareNumber(num, lives):
             print(f"__________Remaining Lives: {lives}__________\n ")
     if lives == 0:
         print('__________YOU LOSE__________\n \n \n ')
+        if mode == 2:
+            print(f'You survived until round {round}!\n ')
+    return lives
+# need to break out smaller methods or split into 2
+
+
+# methods for perpetual mode
+# chooseMode(), addLives(), roundCount(), ??
+
+
+def chooseMode():
+    print("What game mode would you like to play?")
+    mode = input("Enter 1 for Normal or 2 for Perpetual\n(Enter 3 for more information)")
```

Comment:
Any inputs will always need error handling. In this case, if they don't input a valid mode number, you'll want to handle that.